src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -1,15 +1,35 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Truck
- * Do not remove no-field constructor
- */
 public class Truck extends Machine {
```

Comment:
The class Truck should have access modifiers for its fields. Consider making the fields private to follow encapsulation principles.